### PR TITLE
enh(release): Add Centreon Web 21.10.5 RN

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target version

- [ ] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [x] 21.10.x (staging)
- [ ] 22.04.x (next)

### CODE REVIEW

**Review by mdebauchez — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-core.md [hunk 1]
```diff
@@ -16,6 +16,19 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
 
 ## Centreon Web
 
+### 21.10.5
+
+#### Security Fixes
+
+- [Administration] SQL Injections on ACL group listing
+- [Administration] SQL Injections on Knowledge Base configuration form
```
Comment (chgautier) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-core.md:24]:
```suggestion
- [Administration] SQL Injection on Knowledge Base configuration form
```

Comment (chgautier) [reply i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-core.md:24]: MON-12512 is the only ticket for the KB

File: versioned_docs/version-21.10/releases/centreon-core.md [hunk 2]
```diff
@@ -17,6 +17,19 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 21.10.5
+
+#### Security Fixes
+
+- [Administration] SQL Injections on ACL group listing
+- [Administration] SQL Injections on Knowledge Base configuration form
```
Comment (chgautier) [inline versioned_docs/version-21.10/releases/centreon-core.md:25]:
```suggestion
- [Administration] SQL Injection on Knowledge Base configuration form
```

**Review by chgautier — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-core.md [hunk 3]
```diff
@@ -16,6 +16,19 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
 
 ## Centreon Web
 
+### 21.10.5
+
```
Comment (chgautier) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-core.md:20]:
```suggestion

Release date: `March 21, 2022`

```

File: versioned_docs/version-21.10/releases/centreon-core.md [hunk 4]
```diff
@@ -17,6 +17,19 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 21.10.5
+
```
Comment (chgautier) [inline versioned_docs/version-21.10/releases/centreon-core.md:21]:
```suggestion

Release date: `March 21, 2022`

```

**Pull request merged**